tests/test_pipelines_common.py

```diff
@@ -302,7 +302,7 @@ def test_attention_slicing_forward_pass(self):
         output_with_slicing = pipe(**inputs)[0]
 
         max_diff = np.abs(output_with_slicing - output_without_slicing).max()
-        self.assertLess(max_diff, 1e-5, "Attention slicing should not affect the inference results")
+        self.assertLess(max_diff, 1e-3, "Attention slicing should not affect the inference results")
```

Comment:
attention slicing forces different batch size matrix multiplication - there is no way we can guarantee a 1e-5 diff only IMO

Reply:
cc @anton-l